### PR TITLE
Audit article: INFODEV-1655

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-c9d24fc023/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-5e87879d3f/how-to/). :bow:

**Comment by RackerKate** [comment]
@cyrichardson For your review.

**Comment by RackerKate** [comment]
@stephamon For your review.

File: content/cloud-servers/rackspace-cloud-essentials-transferring-your-domain-to-be-served-from-rackspace-cloud.md [hunk 1]
```diff
@@ -1,108 +1,112 @@
...
-#### We are not a domain registrar
+### Frequently asked questions about domain enablement
 
-Rackspace IS a provider of world-class Cloud Hosting services on demand
-backed by our commitment to **Fanatical Support**&reg;. What we are NOT is a
-Domain Registrar like [Namecheap](http://www.namecheap.com/),
+The following FAQs provide answers to frequent questions about domain
+enablement.
+
+#### Rackspace is not a domain registrar
```
Comment (stephamon) [inline content/cloud-servers/rackspace-cloud-essentials-transferring-your-domain-to-be-served-from-rackspace-cloud.md]: I think we should frame this as a question, as this is an FAQ section. So "Is Rackspace a domain registrar?"

File: content/cloud-servers/rackspace-cloud-essentials-transferring-your-domain-to-be-served-from-rackspace-cloud.md [hunk 2]
```diff
@@ -1,108 +1,112 @@
...
+
+Rackspace is a provider of world-class, on-demand Cloud Hosting services
```
Comment (stephamon) [inline content/cloud-servers/rackspace-cloud-essentials-transferring-your-domain-to-be-served-from-rackspace-cloud.md]: If you change the header to a question format, you should update the response to be something like:

No, Rackspace is not a domain registrar. Rackspace is a provider of world-class, on-demand Cloud Hosting services that are backed by our commitment to **Fanatical Support**. 

In order to host a domain from your cloud server, you must first register it at a domain registrar such as Namecheap, Dreamhost, or GoDaddy, and then enable it at Rackspace.

File: content/cloud-servers/rackspace-cloud-essentials-transferring-your-domain-to-be-served-from-rackspace-cloud.md [hunk 3]
```diff
@@ -1,108 +1,112 @@
...
 #### What are Rackspace's name servers?
 
-For Rackspace Cloud services, the name servers you need to use are:
+For Rackspace Cloud services, you need to use the following name servers:
 
-**ns.rackspace.com** & **ns2.rackspace.com**
+- **ns.rackspace.com** and **ns2.rackspace.com**
 
-#### How do I transfer the domain?
+#### How do I start to enable the domain at Rackspace?
```
Comment (stephamon) [inline content/cloud-servers/rackspace-cloud-essentials-transferring-your-domain-to-be-served-from-rackspace-cloud.md]: This header can probably just be "How do I enable the domain at Rackspace?"

File: content/cloud-servers/rackspace-cloud-essentials-transferring-your-domain-to-be-served-from-rackspace-cloud.md (hunk 1 above)
Comment (RackerKate) [reply content/cloud-servers/rackspace-cloud-essentials-transferring-your-domain-to-be-served-from-rackspace-cloud.md]: Great suggestion.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5bc8f48bff/how-to/).

**Pull request merged**